### PR TITLE
 UI Design for FAQs page of the website 

### PR DESCRIPTION
**Pull request by vikhyatsingh123**

### CODE REVIEW

**Comment by vikhyatsingh123** [comment]
active-states
Here is preview of FAQ page

File: frontend/src/components/FAQ/faq.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "./styles/style.css";
+import "./styles/style.scss";
```
Comment (Kajol-Kumari) [inline frontend/src/components/FAQ/faq.js:3]: We aren't using `scss` anywhere so in order to be consistent let's go only  with `css`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@vikhyatsingh123  Please make the requested change

**Comment by Kajol-Kumari** [comment]
Hey @vikhyatsingh123 can you please let us know if you would be able to resolve the requested changes here?

File: frontend/src/components/FAQ/faq.js (hunk 1 above)
Comment (vikhyatsingh123) [reply frontend/src/components/FAQ/faq.js]: ok, I removed scss file

**Review by Kajol-Kumari — COMMENTED** [review]
Hey @vikhyatsingh123 can you please let us know why do we need this file ?`frontend/src/components/FAQ/styles/style.css.map`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Apart from [previous comment](https://github.com/HITK-TECH-Community/Community-Website/pull/45#pullrequestreview-558137341), I am pointing here few other issues that needs to be fixed:

1. Please put all the `FAQ` related code under existing `faq` folder. I can see two folder now, please remove the `FAQ` and keep everything under `faq`.
2. Please follow the structure same as other components i.e keep only the css and js file under the component folder and images etc under `public` folder
3. Remove unnecessary files and changes from the PR.

**Comment by vikhyatsingh123** [comment]
ok, done

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @vikhyatsingh123 thanks for incorporating the requested changes. Please incorporate these requested changes as well. Also in `style.css` the image import is throwing error, please correct it. Also the expandible feature(getting answer after clicking on arrow button) isn't working in the page, please fix that as well.

**Pull request closed without merging**